python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -77,6 +77,18 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         "Ignored if piffPsfConfigYaml is set.",
         default=2,
     )
+    piffBasisPolynomialSolver = pexConfig.ChoiceField[str](
+        doc="Solver to solve linear algebra for "
+        "the spatial interpolation of the PSF. "
+        "Ignore if piffPsfConfigYaml is not None.",
+        allowed=dict(
+            scipy='Default solver using scipy.',
+            cpp='C++ solver using eigen.',
+            jax='JAX solver.',
+            qr='QR decomposition using scipy/numpy.',
```

Comment:
I still don't like the mix of single/double quotes in this PR.

Reply:
I just added double quote.